### PR TITLE
restructure business logic in subfolders

### PR DESCRIPTION
**Pull request by oyo**
started to restructure the business logic in subfolders
for example the folder
`src/state/features/adminRegistration/`
now moved to
`src/features/admin/registration/`

### CODE REVIEW

File: cx-portal/src/assets/locales/en/main.json [hunk 1]
```diff
@@ -359,6 +359,7 @@
       "more": "More",
       "details": "Details",
       "close": "Close",
+      "back": "Back",
       "breadcrumbBack": "Back"
```
Comment (PI-FerasAhmad) [inline cx-portal/src/assets/locales/en/main.json:363]: In my next Ticket I will replace breadcrumbBack with back label

File: cx-portal/src/features/admin/registration/actions.ts [hunk 2]
```diff
@@ -1,33 +1,30 @@
 import { createAsyncThunk } from '@reduxjs/toolkit'
 import RegistrationRequests from 'utils/mockDataSet/registrationRequests.json'
-import { AdminRegistrationApi } from './api'
-import { RegistrationRequestAPIResponse } from './types'
+import { Api } from './api'
+import { name, RegistrationRequestAPIResponse } from './types'
```
Comment (PI-FerasAhmad) [inline cx-portal/src/features/admin/registration/actions.ts:4]: Is _name='admin/registration'_ the route name? 

**${name}/fetchRegistrationRequests** and 
**${name}/fetchRegistrationRequests error**

Comment (oyo) [reply cx-portal/src/features/admin/registration/actions.ts:4]: That name is the path prefix in the Redux store structure. If you have the Redux dev tools installed you can check the names of the actions and the store tree. In theory the actions and store tree paths can have different names. I defined the name in the types for every sub-slice to make sure the actions and the data in the store share the same prefixes.

File: cx-portal/src/assets/locales/en/main.json (hunk 1 above)
Comment (oyo) [reply cx-portal/src/assets/locales/en/main.json:363]: I imagined that - there was only one i18n key on the app details page that didn't look very nice so I added that key again.

**Review by PI-FerasAhmad — APPROVED** [review]
looks good to me

**Pull request merged**